tests/node_converters/pad_test.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import onnx
+import pytest
+
+from tests.utils.common import check_onnx_model
+from tests.utils.common import make_model_from_nodes
+
+
+def _test_pad(
```

Comment:
Add tests for:
1) Different `pad` modes (constant, reflect, edge);
2) Different pad for different dimensions;